DOCKER_IMAGE_VERSION

```diff
@@ -1 +1 @@
-0.15.1-20250603
\ No newline at end of file
+0.16.0-20250613
```

Comment:
Could you only bump the patch number, to `0.15.2-20250613`? I plan to maintain a change log or write release notes for major releases since 0.16.0. So let's not move to 0.16.0 in haste.